dhis-2/dhis-api/src/main/java/org/hisp/dhis/dataanalysis/FollowupValue.java

```diff
@@ -114,4 +121,10 @@ public String getPe()
             ? null
             : PeriodType.getIsoPeriod( PeriodType.getCalendar(), peType, peStartDate );
     }
+
+    @JsonIgnore
+    Period getPeAsPeriod()
```

Comment:
private modifier?

Reply:
Its used in the service but _default_ works I think.

Reply:
It already is default - missed that when I was commenting :D